src/styles/index.scss

```diff
@@ -1,3 +1,221 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  --card-width: 200px;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+
+  &__logo {
+    display: flex;
+
+    &__img {
+      display: block;
+    }
+  }
+}
+
+.nav {
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    text-transform: uppercase;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    position: relative;
+    text-decoration: none;
+    color: black;
+    font-size: 12px;
+    padding: 0;
+    text-align: center;
+    height: 60px;
+    line-height: 60px;
+    margin-left: 20px;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+.is-active {
+  color: #00acdc;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 4px;
+    background-color: #00acdc;
+    border-radius: 8px;
+  }
+}
+
+.card {
+  width: var(--card-width);
```

Comment:
The `.card` class should include a transition for hover effects to ensure changes are smooth over 300ms, as specified in the task requirements【12:3†source】.